### DISCUSSION TITLE
Why 3D ResUNet used in this project trained faster than basicunet and vnet

### DISCUSSION THREAD

**Discussion by Yangtting** [Project-MONAI/tutorials#375] [Q&A]
Hi @wyli @Nic-Ma @ericspod ,
I compared three networks (unet,vnet and basicunet)based on Spleen 3D segmentation with MONAI  using my own dataset. I found out that training unet which has res-block just for 5 hours. But when I replaced it with basic unet and vnet, I trained for about 10h and 55h. I don’t know why these networks look like so similar, but training time is so different. (I didn’t change other things, just replace the networks).  

If some knows, I will appreciate very much. Thank you!

**Comment by Nic-Ma** [comment]
Hi @ericspod @yiheng-wang-nv ,

Could you please help share some comments here?

Thanks in advance.

**Reply by ericspod** [reply]
With the UNet, does the training time go up again if you turn the residual blocks off (`num_res_units=0` in the constructor)? Residual blocks are known to help convergence a lot so these numbers are necessarily off, it's worth looking into published papers in residual connections and what they do, ie. smoothing the loss space you're trying to optimize.
